spec/generator/build_phase/compile_xcdatamodel_build_phase_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+module Xcake
+  module Generator
+    describe CompileXCDataModelBuildPhase do
+
+      before :each do
+        @node = double("Node").as_null_object
+      end
+
+      it "should not able to install folder" do
+        folder_path = 'Folder/'
+
+        allow(@node).to receive(:path).and_return(folder_path)
+        allow(File).to receive(:directory?).with(folder_path).and_return(true)
+
+        expect(CompileXCDataModelBuildPhase.can_install_node(@node)).to be(false)
+      end
+
+      it "should be able to install xcdatamodel foler" do
+        folder_path = 'Model.xcdatamodel'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.